templates/concourse-worker.service.j2

```diff
@@ -4,6 +4,8 @@
 Description=concourse-worker
 Requires=network-online.target
 After=network-online.target
+Wants=concourse-worker-watchdog.service
```

Comment:
no `concourse-worker-watchdog.service` anymore ? should be removed ?

Reply:
We will wait for concourse to fix the issue.  
Right now the PR reflect the second implementation where the watchdog is a sub-process in the background of the main concourse-worker process (\`NotifyAccess=main\`) but I prefer my first implementation where the watchdog was a separated process (\`NotifyAccess=all\`)

Reply:
currently it is a mix of both ? Sub-process is used but ext service is referenced here.

Just a reminder note, for ext service this part will be missing: https://github.com/cycloidio/ansible-concourse/pull/7/files#diff-2f8071c99c715eb05ff6fabd9fed7dd5R15

Reply:
> currently it is a mix of both ? Sub-process is used but ext service is referenced here.

Yes, I did a bunch of tests for different implementations, I must have missed some old references ^^

> Just a reminder note, for ext service this part will be missing: https://github.com/cycloidio/ansible-concourse/pull/7/files#diff-2f8071c99c715eb05ff6fabd9fed7dd5R15

Good catch, I will update this PR as soon as possible to represent the ideal setup if Concourse will ever fix the issue blocking it.